### PR TITLE
Exteded telemetry fetching

### PR DESCRIPTION
**Pull request by lperryp**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32421

## Description
Improved collection:

- Collect also close notes: remove numeric strings, and hash out-of-vocabulary words
- Collect owner, hash if not empty
- Check for other possible fields which might indicate whether the incident was closed automatically
- Subject embedding
- Collect whether a subject contains FW (for detecting forwarded mails)
- preprocess email subject (remove [] prefixes )

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py [hunk 1]
```diff
@@ -155,6 +156,14 @@ class ShortTextException(Exception):
                      "tinyurl.ms", "tip.pe", "ty.by"}
 
 
+def hash_value(simple_value):
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py:159]: plz use the has in common server python, better to use the same hash function as we used in the telemetry

**Review by erezh31 — APPROVED** [review]
looks good small comment plz change that

**Comment by kirbles19** [comment]
@lperryp I made a bunch of changes to the RN. Please check for accuracy.

**Comment by lperryp** [comment]
> @lperryp I made a bunch of changes to the RN. Please check for accuracy.

Thanks!

**Pull request merged**